doc/source/getting_started/running_mapdl.rst

```diff
@@ -173,6 +173,63 @@ the output of MAPDL within Python and can be used to debug why MAPDL
 isn't launching.  Output will be limited on Windows due to the way
```

Comment:
```suggestion
isn't launching.  Output is limited on Windows due to the way
```

Reply:
@germa89 Although Vale isn't implemented here, I'm manually editing according to Google developer style guides. They recommend avoiding future tense and using present tense instead.